### PR TITLE
ivykis 0.41 (new formula)

### PR DESCRIPTION
**Pull request by MrAnno**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/ivykis.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class Ivykis < Formula
+  desc "Async I/O-assisting library"
+  homepage "http://libivykis.sourceforge.net"
+  url "https://github.com/buytenh/ivykis/archive/v0.41.tar.gz"
```
Comment (woodruffw) [inline Formula/ivykis.rb]: Let's fetch it from here instead: https://sourceforge.net/projects/libivykis/files/0.41/

File: Formula/ivykis.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+class Ivykis < Formula
+  desc "Async I/O-assisting library"
+  homepage "http://libivykis.sourceforge.net"
```
Comment (woodruffw) [inline Formula/ivykis.rb]: That page leads to this, which looks like the real homepage: https://sourceforge.net/projects/libivykis/

**Review by woodruffw — CHANGES_REQUESTED** [review]

**Comment by woodruffw** [comment]
Debian et al. provide this, so I think we can ignore the notability warning here. It'll also disappear once the `url` link is moved to sourceforge.

**Comment by MrAnno** [comment]
Thanks for the review.

I've updated the PR based on your notes. I had some issues with the sha256 sum because of the SourceForge mirrors, that was the reason I switched to the GitHub URL.

**Comment by woodruffw** [comment]
No problem!

> I had some issues with the sha256 sum because of the SourceForge mirrors, that was the reason I switched to the GitHub URL.

Could you elaborate? Was the `sha256` varying across mirrors?

**Comment by MrAnno** [comment]
> Could you elaborate? Was the sha256 varying across mirrors?

Sorry, I was wrong. All the available SourceForge mirrors have the same checksum, I just mixed up the GitHub and the SourceForge tarball, so I entered a wrong sha256 value.

**Review by woodruffw — APPROVED** [review]

**Comment by woodruffw** [comment]
Thanks @MrAnno, and congratulations on your first contribution/formula! :tada:

**Pull request closed without merging**